src/analysis/normed_space/star/spectrum.lean

```diff
@@ -188,4 +188,15 @@ noncomputable instance : star_hom_class F A ℂ :=
 noncomputable def _root_.alg_hom_class.star_alg_hom_class : star_alg_hom_class F ℂ A ℂ :=
 { .. hF, .. weak_dual.complex.star_hom_class }
 
+omit hF
+
+namespace character_space
+
+noncomputable instance : star_alg_hom_class (character_space ℂ A) ℂ A ℂ :=
+{ coe := λ f, (f : A → ℂ),
+  map_star := λ f, map_star f,
+  .. character_space.alg_hom_class }
```

Comment:
```suggestion
alg_hom_class.star_alg_hom_class
```

Reply:
I'm left the `coe` field as is to ensure proper definitional unfolding, and while I'm at it I'm fixing `alg_hom_class.star_alg_hom_class` in the same way.